scripts/linters/ts_ignore_exceptions.json

```diff
@@ -0,0 +1,145 @@
+{
+  "core/templates/domain/improvements/NeedsGuidingResponsesTaskObjectFactorySpec.ts": [
+    "task_type: '???',"
+  ],
+  "extensions/interactions/LogicProof/static/js/tools/logic-demo-test.controller.spec.ts": [
+    "properties than entries according with lint settings.",
```

Comment:
Firstly, I am a bit confused. In https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html it says "ts-ignore suppresses errors on the following line". But the "following line" here is a comment, so surely it has no errors and the ts-ignore can be removed?

If the line that is being suppressed is the line of code, then the thing quoted in this file should be the line of code (i.e. the line that is being suppressed) and the check should be modified accordingly.

(Also, random aside: below this line in the original file, there is a typo: ore --> more.)

Reply:
done